.github/workflows/git-issues-jira-automation.yaml

```diff
@@ -0,0 +1,18 @@
+name: Git Issues Jira Automation Pipeline
+on:
+  issues:
+    types: [opened, closed]
+  issue_comment:
+    types: [created]
+
+env:
+  AKEYLESS_JIRA_BASE_URL: ${{ secrets.AKEYLESS_JIRA_BASE_URL }}
```

Comment:
instead of duplicating the same secrets across each repo , why aren't you using Organization secrets?